.github/workflows/agw.pr_main.yml

```diff
@@ -44,3 +42,10 @@ jobs:
           sudo apt install libsystemd-dev
           set -eux
           tox -e unit
+
+  build-snap:
+    name: Build Magma AGW snap
+    runs-on: telco-maas-host
```

Comment:
Same as above - build can run on any linux box, but we also have to be able to send the snap to the test env. Since test env is in our lab, which can't be accessed from outside, build has to be done inside the lab too.